docs/throttle-design.md

```diff
@@ -107,5 +107,7 @@ A final bucket is valuable to put some ceiling on the queries-per-second (qps) a
             ]
         }
 ```
+As of Hedera release 0.20 an additional mechanism for throttling contract-related transactions (`ContractCall`, `ContractCallLocal`, `ContractCreate`) was introduced. It involves a `frontend` (gRpc incoming transactions to the node) maximum gas per second throttle and a `consensus` (backend upon start of the actual execution of the transaction) maximum gas per second allowed property. These are configured by the `totalAllowedGasPerSecFrontend` and `totalAllowedGasPerSecConsensus` properties. 
```

Comment:
I was about to say this should reference the throtting hip, but the editors still haven't reviewed it and given it a number :(